tests/app/helpers/conftest.py

```diff
@@ -192,3 +198,28 @@ def expected_footer_nisra_theme():
             "alt": "NISRA - Northern Ireland Statistics and Research Agency",
         },
     }
+
+
+@pytest.fixture()
+def session_data():
```

Comment:
There is already one `session_data` at [test level](https://github.com/ONSdigital/eq-questionnaire-runner/blob/master/tests/app/conftest.py#L78-L91) , may be re-use that

Reply:
Done.